main.py

```diff
@@ -15,7 +15,7 @@
     "(CODE_OF_CONDUCT.md)": "(development/#tides-code-of-conduct)",
     "(CLA.md)": "(development/#tides-contributor-license-agreement)",
     "CONTRIBUTING.md)": "development/)",
-    "<LICENSE>": "[LICENSE](https://github.com/TIDES-transit/TIDES/blob/main/LICENSE)",
+    "<LICENSES>": "[LICENSES](https://github.com/TIDES-transit/TIDES/blob/main/LICENSES)",
```

Comment:
This link currently fails on the preview because it's coded to `main`, but I don't think that matters, and I see you have another PR addressing linking.